common/ts/publish-task.ts

```diff
@@ -28,13 +31,30 @@ export default async function publishTask(endpointType: EndpointType) {
     taskReports.map(taskReport => getReportForTask(taskReport, metrics, endpoint, timeoutSec))
   );
 
+  if (globalQualityGateStatus === '') {
```

Comment:
Does this mean we're treating _unknown_/_no analysis_ the same as _succeeded_?

Reply:
Yes, but you're right, i will add warn and unknown in this logic. Since the gate is yes / no, there will be no other kind of possible solution for the moment.